### ISSUE TITLE
Issues and inconsistencies with custom values

### ISSUE THREAD

**Issue by jeremyzilar** [uswds/uswds#2925]
I am looking to set a few custom values for use in my project.
Specifically, I need a few custom widths that are not defines in the current utilities.

So I went looking to implement the "custom values" found in the `_uswds-theme-utilities.scss` file, and are documented in the advanced section of each of the utilities pages.

The issues I am seeing are:
- [ ] Inconsistent naming and descriptions between uswds setting files and the documentation on the website
- [ ] Unable to replicate the example given in the docs

### 1
The docs say that the file I should be editing is the `_project-utilities-settings.scss` file:
> Customize and refine any utility family’s specific values by editing the `$[utility_family]-manual-values` and `$[utility_family]-palettes variables` in `_project-utilities-settings.scss`.

However in the file that is in BETA 5 is named `_uswds-theme-utilities.scss`.

### 2
The docs example uses
```
$background-color-custom-values: (
  'paper': $base-lightest,
);
```
when it should be:
```
$background-color-manual-values: (
  'paper': $base-lightest,
);
```

### 1

When I use the example given, 
```
$background-color-manual-values: (
  'paper': $base-lightest,
);
``` 
I get the following error — `Undefined variable: "$base-lightest".` 

### 2
When I use
```
$background-color-manual-values: (
  'paper': '#333333',
);
``` 
it builds fine, and `bg-paper` is available as a utility class.
However, when I try to use it as a include in a scss file,
```
.example{
  @include u-bg('paper');
}
```
I am getting the error — `'paper' is not a valid color token.`

I guess I expected to be able use this value anywhere in the project.
_This may not be a bug, but more my lack of understanding of what is possible._

**Comment by thisisdano** [comment]
Thanks for submitting this issue. Custom values are a little tricky and the documentation wasn't even accurate. 😬 I've fixed the documentation to use the proper variables and file names. 

A couple things:
1. Custom values can't use variables as values. Settings files are included before variables are even defined, so custom values can't use them.
2. Custom values only apply to utilities — they aren't tokens available to component USWDS

I've updated the docs to use an accurate example:

```
$width-manual-values: (
  'tab': 4ch,
);
```

Fixes are going in https://github.com/uswds/uswds-site/pull/726
[Preview](https://federalist-proxy.app.cloud.gov/preview/uswds/uswds-site/dw-font-do-si-do/utilities/height-and-width/#advanced-settings)
